.gitignore

```diff
@@ -58,5 +58,5 @@ data/
 
 #generated applications and source
 priv/apps/child_chain_api
-priv/apps/watchers_informational_api
+priv/apps/watcher_informational_api
```

Comment:
this still retains the long name. Change to `watcher_info_api`?

Reply:
This one is autogenerated by swagger for the specs test so I left them as they are.

Reply:
Found it with help from @InoMurko.

Fixed (hopefully without anything failing) in b6ac62b